kubetest/main.go

```diff
@@ -246,6 +248,23 @@ func getDeployer(o *options) (deployer, error) {
 		return dind.NewDeployer(o.kubecfg, o.dindImage, control)
 	case "gke":
 		return newGKE(o.provider, o.gcpProject, o.gcpZone, o.gcpRegion, o.gcpNetwork, o.gcpNodeImage, o.gcpImageFamily, o.gcpImageProject, o.cluster, &o.testArgs, &o.upgradeArgs)
+	case "eks":
+		cfg := eksconfig.NewDefault()
```

Comment:
I feel like all the eksconfig call can happen inside the deployer, so main won't have eks dependencies. wdyt

(In the future we will want to provide a deployer register function, and you can move all deployer codes to your k-sigs repo)

Reply:
> you can move all deployer codes to your k-sigs repo

This sounds awesome!

Yes, let me move everything to "kubetest/eks" package, so main.go does not import anything other than  "kubetest/eks".